pkg/service/presentation/model.go

```diff
@@ -0,0 +1,35 @@
+package presentation
+
+import (
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/TBD54566975/ssi-sdk/util"
+)
+
+const (
+	Version1 string = "1.0"
```

Comment:
Where is this used?

Reply:
+1 we are using V2 of the spec or is this for something else?